doc/api/esm.md

```diff
@@ -95,8 +95,26 @@ When loaded via `import` these modules will provide a single `default` export
 representing the value of `module.exports` at the time they finished evaluating.
 
 ```js
-import fs from 'fs';
-fs.readFile('./foo.txt', (err, body) => {
+// foo.js
+module.exports = { one: 1 };
+
+// bar.js
+import foo from './foo.js';
+foo.one === 1; // true
+```
+
+Builtin modules will provide named exports of their public API, as well as a
+default export which can be used for, among other things, modifying the named
+exports.
+
+```js
+import EventEmitter from 'events';
+const e = new EventEmitter();
+```
+
+```js
+import { readFile } from 'fs';
+readFile('./foo.txt', (err, body) => {
```

Comment:
Nit: it seems a bit unexpected to name file content `body` (no `body` in `fs` doc), it seems more natural for `http(s)` docs. However, I can miss some conventions)

Reply:
does `src`/`source` work?

Reply:
Yes, it seems more neutral to me.